src/UI/Implementation/Component/Panel/Renderer.php

```diff
@@ -0,0 +1,121 @@
+<?php
+
+/* Copyright (c) 2016 Timon Amstutz <timon.amstutz@ilub.unibe.ch> Extended GPL, see docs/LICENSE */
+
+namespace ILIAS\UI\Implementation\Component\Panel;
+
+use ILIAS\UI\Implementation\Render\AbstractComponentRenderer;
+use ILIAS\UI\Renderer as RendererInterface;
+use ILIAS\UI\Component;
+
+/**
+ * Class Renderer
+ * @package ILIAS\UI\Implementation\Component\Panel
+ */
+class Renderer extends AbstractComponentRenderer {
+	/**
+	 * @inheritdocs
+	 */
+	public function render(Component\Component $component, RendererInterface $default_renderer) {
+		/**
+		 * @var Component\Panel\Panel $component
+		 */
+		$this->checkComponent($component);
+
+		if ($component instanceof Component\Panel\Standard) {
+			/**
+			 * @var Component\Panel\Standard $component
+			 */
+			return $this->render_standard($component, $default_renderer);
+		} else if($component instanceof Component\Panel\Sub) {
+			/**
+			 * @var Component\Panel\Sub $component
+			 */
+			return $this->render_sub($component, $default_renderer);
+		}
+		/**
+		 * @var Component\Panel\Report $component
+		 */
+		return $this->render_report($component, $default_renderer);
+	}
+
+	/**
+	 * @param Component\Panel\Standard $component
+	 * @param RendererInterface $default_renderer
+	 * @return string
+	 */
+	protected function render_standard(Component\Panel\Standard $component, RendererInterface $default_renderer)
+	{
+		$tpl = $this->getTemplate("tpl.panel.html", true, true);
```

Comment:
Use a separate template for standard panel and report panel please. This would allow our designers to change them independently.